config.json

```diff
@@ -14,6 +14,7 @@
   "db": {
     "host": "localhost",
     "port": 5432,
+    "schema": "public",
```

Comment:
This change should also be reflected to 

https://github.com/LiskHQ/lisk/blob/2c0398a61bc350f1f9ec6acef29686c1edf370d3/schema/config.js#L62-L64

So that the config schema can be verified.

Reply:
The thing is, it is not needed there. The parameter is redundant for the connection itself. It is only there to be picked up by the SQL loader, and that's it!

Reply:
For any usage, if we add something to config.json, we add its corresponding schema entry as well. Also make it required attribute as well, so if someone remove it from the config.json before starting the lisk, it should not start. Because if we don't have schema parameter, all queries will be malformed.